### ISSUE TITLE
How do I link boost-spirit in cmake?

### ISSUE THREAD

**Issue by am2222** [microsoft/vcpkg#9434]
How can I ink boost::spirit lib file in cmake?
I am using this cmake file

```
find_package(Boost REQUIRED
             COMPONENTS filesystem system regex program_options)
target_link_libraries(mapnik PUBLIC
    ${thirdparty_libraries}
    Boost::boost Boost::filesystem Boost::system Boost::regex Boost::program_options
    Freetype::Freetype
    ICU::uc
    PNG::PNG TIFF::TIFF
    ZLIB::ZLIB)

```
which linkes other boost components, but when I try to add Boost::spirit to target_link_libraries it cannot find it. In my vcpkg compiled library lists I can find boost-spirit as following
```
vcpkg install boost-spirit:x64-windows
The following packages are already installed:
    boost-spirit[core]:x64-windows
Starting package 1/1: boost-spirit:x64-windows
Package boost-spirit:x64-windows is already installed
Elapsed time for package boost-spirit:x64-windows: 4.236 ms

Total elapsed time: 4.441 ms
```

If I use 
```
find_package(Boost REQUIRED
             COMPONENTS spirit)
```
Cmake can not find this component. The question is that which boost component should I use to be able to link `boost-spirit` ?
Thanks

**Comment by am2222** [comment]
I guess problem is with a bug in boos 1.71 which will be fixed if I update boost to 1.72

**Comment by BraunPhilipp** [comment]
boost spirit is header-only; no explicit linking is needed.
